helm-charts/HPA.md

```diff
@@ -100,60 +100,37 @@ used model, data type and image versions, the specified resource values may need
 
 ### Post-install
 
-Above step created custom metrics config for Prometheus-adapter suitable for HPA use.
+Above step created custom metrics config for Prometheus-adapter, suitable for HPA use.
```

Comment:
The above step created a custom metrics configuration for Prometheus adapter, suitable for HPA use.